services/web/server/src/simcore_service_webserver/exporter/archiving.py

```diff
@@ -86,21 +73,17 @@ async def zip_folder(project_id: str, input_path: Path, no_compression=False) ->
             f"Cannot archive because file already exists '{str(zip_file)}'"
         )
 
-    command_args = [
-        "zip",
-        "-0",
-        "-r",
-        str(zip_file),
-        project_id,
-    ]
-    if no_compression:
-        command_args.remove("-0")
-
-    await _wrap_create_subprocess_exec(
-        command_args=command_args,
-        cwd_path=input_path.parent,
-        fail_message=f"Could not create archive {str(zip_file)}",
-    )
+    try:
+        await run_in_process_pool(
+            shutil.make_archive,  # callable
+            Path(input_path.parent) / project_id,  # base_name
+            "zip",  # format
+            input_path.parent,  # root_dir
+            project_id,  # base_dir
+        )
+    except Exception as e:
+        log.exception(e)
+        raise ExporterException(str(e)) from e
```

Comment:
one thing:
with @pcrespov we recently found out that catching general Exceptions like this leads to a lot of issues regarding in particular CancelledError.
Cancellation might arise when a request is cancelled by the client (in your case might be through the API or the GUI). A cancellation is ok, it should stop the zipping process and be re-raised. in this case I am not sure this will not be suddenly transformed into a real error?

Reply:
So this will catch the exception log it, and then transform it in an `ExporterException` which is what.
This transforms any type of `Exception` into an `ExporterException` which is propagated and transformed into a 4xx response.

Reply:
well which is probably not correct in the case of a cancellation from the client you do not have to return anything. just cancel the whole task. @pcrespov has maybe some more insights here.

Reply:
I am talking about request cancellation. Maybe we are talking about two different things then.
Maybe catching the CancelError and raising it again before trapping all generic exceptions would allow for a better behaviour.

Reply:
I guess you probably log the exporterexcerption as an error in the log don't you?
if the user cancels the request, then it should be handled, maybe logged as such but it should not deserve a big error exception thingy that polute the logs.
and I agree with the your latest comment. probably then handling it before going to the broader one.

Reply:
OK, addressed both concerns